python/cudf/udf_cpp/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ rapids_find_package(
   INSTALL_EXPORT_SET udf-exports
 )
 
-# The below in vendored/inspired by legate dataframe
+# The below in inspired by legate dataframe
 execute_process(
   COMMAND "python" -c "from numba.cuda.runtime.nrt import get_include; print(get_include())"
```

Comment:
Is it possible to amend this so that it can try importing `get_include` from `numba.cuda.runtime.nrt` and if that fails with an `ImportError`, to import it from `numba.cuda.memory_management.nrt`? This is because we accidentally shadowed the `numba.cuda.runtime` module with the `numba.cuda.runtime` package, so we need to rename the package. (I have this fix in https://github.com/NVIDIA/numba-cuda/pull/178 - it causes more of an issue for the simulator than the "normal" target).

Reply:
Would it be better to get the numba PR in and change it to just always be `memory_managment` ? We're now targeting 25.08 for this PR so there's some wiggle room

Reply:
Thanks for looking at #178. I need to do a couple of fixups there due to recent changes in `main` but it's substantially complete.